### PR TITLE
Pa 11039 es schema update

### PR DESCRIPTION
**Pull request by lorinbeer**
adds expanded job template spec and job spec

### CODE REVIEW

File: spec.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+	listValueId          *string
+	selectedListValues   map[string]bool
+	selectedListValueIds map[string]bool
+}
+
+type DeviationsDocument struct{}
+
+// mirrors mothership/models/job_template.go::JobTemplate, but can/should be able to diverge independently
+type JobTemplateDocument struct {
+	id                string                        `model:"Id"`
```
Comment (taiyangc) [inline spec.go]: should be the other way?

```Id string `model:"id"``` 

lower case fields cannot be accessed from other packages

File: spec.go [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+	created_at        int64                         `model:"CreatedAt"`
+	updated_at        int64                         `model:"UpdatedAt"`
+	archived_at       int64                         `model:"ArchivedAt"`
+	published_version int32                         `model:"PublishedVersion"`
+	public_version    int32                         `model:"PublicVersion"`
+	version           int32                         `model:"Version"`
+	title             string                        `model:"Title"`
+	description       string                        `model:"Description"`
+	status            string                        `model:"Status"`
+	metadata          TemplateMetadataEntryDocument `model:"MetadataEntries"`
```
Comment (taiyangc) [inline spec.go]: `map[string]*TemplateMetadataEntryDocument`?

File: spec.go [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+	flags             int32                         `model:"Flags"`
+	typ               int32                         `model:"Type"`
+}
+
+// mirrors mothership/models/job.go::Job, but can/should be able to diverge independently
+type JobDocument struct {
+	id                    string `model:"Id"`
+	team_id               string `model:"TeamId"`
+	lookup_id             int64  `model:"lookp_id"`
```
Comment (taiyangc) [inline spec.go]: typo

File: spec.go (hunk 1 above)
Comment (lorinbeer) [inline spec.go]: ok, that's a little frustrating. I'm going to dig down. I like it this way as it simplifies interacting with the db and models in mothership.

File: spec.go (hunk 2 above)
Comment (lorinbeer) [inline spec.go]: updated

File: spec.go (hunk 3 above)
Comment (lorinbeer) [inline spec.go]: updated

**Pull request merged**